src/main/resources/edu/ucsd/sbrg/polisher/Messages_de.xml

```diff
@@ -162,6 +164,7 @@
     <entry key="VAL_ERROR">Keine SBML Validierung möglich. Prozess mit Fehlern beendet.</entry>
     <entry key="VERSION_NR_MISSING">Die Versionsnummer für das Model fehlt</entry>
     <entry key="WRITE_COMBINE_ARCHIVE">Schreibe COMBINE Archiv {0}</entry>
+    <entry key="WRITE_DIR_TO_FILE_ERROR">Cannot write inputs from directory {0} to a output file {1}.</entry>
```

Comment:
If a translation is not known (of course, German classes are not required for GSoC) just leave out the corresponding entry in the text file. The `ResourceManager` will automatically pick the default message, which is going to be the one from the English resource file.

It is easier to figure out later on, which `String`s are missing in a translation file than going through them to identify the ones that are in the wrong language.